Storage/SQL/BrowserDB.swift

```diff
@@ -99,7 +113,7 @@ open class BrowserDB {
 
     // Checks if the database schema needs created or updated and acts accordingly.
     // Calls to this function will be serialized to prevent race conditions when
-    // creating or updating the schema.
+    // creating or updating the schema. (Exposed for testing purposes only.)
     func prepareSchema(_ schema: Schema) -> DatabaseOpResult {
```

Comment:
The only instance variables used by this function are `log` and `db`. Making this static will help to cut the cycle.